packages/integration-oauth/src/providers/apple.ts

```diff
@@ -0,0 +1,128 @@
+import * as fs from "node:fs/promises";
```

Comment:
Can't use Node dependencies. I'll recommend letting the user read the file and pass the string content. (Need to update the docs too).

Reply:
fixed